specutils/spectra/spectral_axis.py

```diff
@@ -0,0 +1,95 @@
+import warnings
+
+import astropy.units as u
+from astropy.utils.decorators import lazyproperty
+import numpy as np
+
+from .spectral_coordinate import SpectralCoord
+
+
+__all__ = ['SpectralAxis']
+
+# We don't want to run doctests in the docstrings we inherit from Quantity
+__doctest_skip__ = ['SpectralAxis.*']
+
+class SpectralAxis(SpectralCoord):
+    """
+    Coordinate object representing spectral values corresponding to a specific
+    spectrum. Overloads SpectralCoord with additional information (currently
+    only bin edges).
+
+    Parameters
+    ----------
+    value : ndarray or `~astropy.units.Quantity` or `SpectralCoord`
+        Spectral axis data values.
+    unit : str or `~astropy.units.Unit`
+        Unit for the given data.
+    observer : `~astropy.coordinates.BaseCoordinateFrame` or `~astropy.coordinates.SkyCoord`, optional
+        The coordinate (position and velocity) of observer.
+    target : `~astropy.coordinates.BaseCoordinateFrame` or `~astropy.coordinates.SkyCoord`, optional
+        The coordinate (position and velocity) of observer.
+    radial_velocity : `~astropy.units.Quantity`, optional
+        The radial velocity of the target with respect to the observer.
+    redshift : float, optional
+        The redshift of the target with respect to the observer.
+    doppler_rest : `~astropy.units.Quantity`, optional
+        The rest value to use for velocity space transformations.
+    doppler_convention : str, optional
+        The convention to use when converting the spectral data to/from
+        velocity space.
```

Comment:
```suggestion
```

Since these exist in the base class and are handled there, there is no need to copy them here in the subclass.

Reply:
Removed.